package.json

```diff
@@ -1469,7 +1470,6 @@
     "@types/selenium-webdriver": "^4.1.21",
     "@types/semver": "^7",
     "@types/set-value": "^2.0.0",
-    "@types/sharp": "^0.30.4",
```

Comment:
No, I just think we were not using them anywhere. In the past sharp was being required without being listed in the package.json (it was installed as a dependency of other dependency we were using). Because of that I suspect anyone have added the types because of the require of sharp. Then with the declaration of sharp in the package.json and the upgrade into a recent version (where types are shipped along with the node module itself) there is no need to keep the types here :)